libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -295,14 +302,68 @@ int32_t AP_BattMonitor::pack_capacity_mah(uint8_t instance) const
     }
 }
 
- /// exhausted - returns true if the voltage remains below the low_voltage for 10 seconds or remaining capacity falls below min_capacity_mah
-bool AP_BattMonitor::exhausted(uint8_t instance, float low_voltage, float min_capacity_mah)
+void AP_BattMonitor::check_failsafes(void)
```

Comment:
This looks ok, but I think anyone else who visits this bit of code will appreciate a comment with a high-level rundown of what is going on here.